userspace/libscap/scap.h

```diff
@@ -199,6 +199,7 @@ typedef struct scap_threadinfo
 	uint64_t sid; ///< The session id of the process containing this thread.
 	char comm[SCAP_MAX_PATH_SIZE]; ///< Command name (e.g. "top")
 	char exe[SCAP_MAX_PATH_SIZE]; ///< argv[0] (e.g. "sshd: user@pts/4")
+	char exepath[SCAP_MAX_PATH_SIZE]; ///< full executable path
```

Comment:
After testing this with file output it seems that the execve enter event is missing the calling process name.

File capture:
`2309 09:03:45.826018329 3 <NA> (85726) > execve filename=/bin/ls`
Live capture:
`54585 09:10:52.693714381 0 bash (85892) > execve filename=/bin/ls`

While the live capture is fine.
How can I fix this?